src/snowflake/cli/plugins/snowpark/models.py

```diff
@@ -13,6 +14,30 @@ class PypiOption(Enum):
     ASK = "ask"
 
 
+class RequirementType(Enum):
+    FILE = "file"
+    PACKAGE = "package"
+
+
+class Requirement(requirement.Requirement):
+    @classmethod
+    def parse_line(cls, line: str) -> Requirement:
+        if len(extras := line.split(";")) > 1:
+            line = extras[0]
+        result = super().parse_line(line)
+
+        if len(extras) > 1:
+            result.extras = extras[1]
```

Comment:
Is there a possibility that there will be more than 1 item in extra part?
`hello;something;more`

Or it will always be `something; more`?